README.rst

```diff
@@ -2,20 +2,38 @@
 MPI for Python
 ==============
 
-.. image::  https://github.com/mpi4py/mpi4py/workflows/ci/badge.svg?branch=master
-   :target: https://github.com/mpi4py/mpi4py/actions/
-.. image::  https://readthedocs.org/projects/mpi4py/badge/?version=latest
-   :target: https://mpi4py.readthedocs.io/en/latest/
-.. image::  https://dev.azure.com/mpi4py/mpi4py/_apis/build/status/ci?branchName=master
-   :target: https://dev.azure.com/mpi4py/mpi4py/_build
-.. image::  https://ci.appveyor.com/api/projects/status/whh5xovp217h0f7n?svg=true
-   :target: https://ci.appveyor.com/project/mpi4py/mpi4py
-.. image::  https://circleci.com/gh/mpi4py/mpi4py.svg?style=shield
-   :target: https://circleci.com/gh/mpi4py/mpi4py
-.. image::  https://codecov.io/gh/mpi4py/mpi4py/branch/master/graph/badge.svg
-   :target: https://codecov.io/gh/mpi4py/mpi4py
-.. image::  https://scan.coverity.com/projects/mpi4py-mpi4py/badge.svg
-   :target: https://scan.coverity.com/projects/mpi4py-mpi4py
+-  .. image:: https://github.com/mpi4py/mpi4py/workflows/ci/badge.svg?branch=master
```

Comment:
I don't quite like using list, the bullet point followed by badges looks a bit awful IMO. 
What if you just add a blank line as a separator? That should be enough to force the related badges on its own line.

Reply:
Tried that, but unfortunately that didn't seem to work with RST formatting.